### PR TITLE
fix(j-s): Incident Description Autofill

### PR DESCRIPTION
**Pull request by gudjong**
# Incident Description Autofill

[Umferðarlagabrot - ölvunarakstur](https://app.asana.com/0/1199153462262248/1203865140443261/f)

## What

- Fixes incident description autofill removing duplicate text.
- Fixes drivers licence revoke request making it always available.

## Why

- Verified bugs.

## Screenshots / Gifs

image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/Indictment.tsx [hunk 1]
```diff
@@ -264,14 +257,6 @@ const Indictment: React.FC = () => {
 
   useOnceOn(isCaseUpToDate, initialize)
 
-  const enableDriversLicenseSuspension = useMemo(
```
Comment (unakb) [inline apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/Indictment.tsx:267]: Please remove useMemo import as well

File: apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/IndictmentCount.tsx [hunk 2]
```diff
@@ -156,35 +157,30 @@ function getIndictmentDescriptionReason(
...
-                strings.trafficViolationIncidentDescriptionDrugsDrivingPrefixAutofill,
+                strings.incidentDescriptionDrugsDrivingPrefixAutofill,
               )) +
           formatMessage(
-            strings.trafficViolationIncidentDescriptionPrescriptionDrugsDrivingAutofill,
+            strings.incidentDescriptionPrescriptionDrugsDrivingAutofill,
           )
         break
     }
+    console.log(acc)
```
Comment (unakb) [inline apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/IndictmentCount.tsx:183]:
```suggestion
```

**Review by unakb — CHANGES_REQUESTED** [review]
A few minor things, also one question. I noticed in text in the design docs that before the list relevant substances it says "í blóðsýni mældist ... ". Is the text there just out of date or do we need to add it into the relevantSubstances reason string

File: apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/IndictmentCount.tsx [hunk 3]
```diff
@@ -143,6 +143,7 @@ function getIndictmentDescriptionReason(
   substances: SubstanceMap,
   formatMessage: IntlShape['formatMessage'],
 ) {
+  console.log(offenses)
```
[the remark this answers is not in the document]
Comment (gudjong) [reply apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/IndictmentCount.tsx:146]: Sloppy 😊

**Comment by gudjong** [comment]
> A few minor things, also one question. I noticed in text in the design docs that before the list relevant substances it says "í blóðsýni mældist ... ". Is the text there just out of date or do we need to add it into the relevantSubstances reason string?

Good point. I will investigate and fix.

**Review by unakb — APPROVED** [review]

**Pull request merged**